### PR TITLE
Chart kit Additions

### PR DESCRIPTION
**Pull request by jasperfurniss**
This PR adds a few additional to props to both the Y and X axis of the bar graph, enabling Jon to move forward with solar and style the bar graphs with our kit. It allows us to be able to define the tick interval, a min and max value for the labels on the axis, as well as custom categories. Looking for feedback!

### CODE REVIEW

File: app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb [hunk 1]
```diff
@@ -14,6 +14,14 @@
         subtitle: "<%= object.subtitle %>",
         axisTitle: "<%= object.axis_title %>",
         pointStart: <%= object.point_start %>,
+        xCategories: <%= object.x_categories.any? ? object.sanitized_x_categories : "undefined" %>,
```
Comment (terryfinn) [inline app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb]: We just push this logic up into the view object?

File: app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
+        xMin: <%= object.x_min %>,
+        xMax:<%= object.x_max %>,
+        xTickInterval: <%= object.x_tick_interval %>,
+        yCategories: <%= object.y_categories.any? ? object.sanitized_y_categories : "undefined"  %>,
```
Comment (terryfinn) [inline app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb]: We just push this logic up into the view object?

**Comment by jasoncypret** [comment]
@jasperfurniss are these required now? Seems like we should make that optional

File: app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb (hunk 2 above)
Comment (xjunior) [reply app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb]: I think that all of this chart object could go into the view model.

**Comment by jasoncypret** [comment]
@kre8sions Are you picking up this work, or starting a new PR?

**Comment by thestephenmarshall** [comment]
@jasperfurniss any updates here?

**Pull request closed without merging**